templates/downloads/os_list.html

```diff
@@ -38,8 +38,12 @@ <h2>Stable Releases</h2>
                     {% for r in releases %}
                     <li>
                         <a href="{{ r.get_absolute_url }}">{{ r.name }} - {{ r.release_date|date }}</a>
-                        {% if os.slug == 'windows' and r.name >= 'Python 3.5' %}
-                        <p><strong>Note that {{ r.name }} <em>cannot</em> be used on Windows XP or earlier.</strong></p>
+                        {% if os.slug == 'windows' %}
+                            {% if r.name >= 'Python 3.9' %}
```

Comment:
Wouldn't this return `False` if `r.name` is `'Python 3.10'`?

Reply:
Probably!  But that's just what the existing code does.  I was going to open a separate issue to go through the whole repo to fix all similar version checks prior to 3.10.0 a year from now.  Do you have any idea if there are other such version checks?

Reply:
Yep!
```pycon
>>> 'Python 3.10' >= 'Python 3.9'
False
```

A quick grep doesn't show any more of this pattern:

```console
$ git grep -E "[23]\..* [<>][=]? "
$ git grep -E " [<>][=]? .*[23]\."
static/sass/_layout.scss:        & > .column { width: 33.3333%; }
templates/downloads/os_list.html:                        {% if os.slug == 'windows' and r.name >= 'Python 3.5' %}
```

Running https://github.com/asottile/flake8-2020/ doesn't find anything:

```console
$ pip install -qU flake8-2020 && flake8 --select YTT
$
```

Still worth a more thorough review.

Reply:
If anyone wants to come up with a proper version test, please feel free to do so! My immediate concern is the imminent 3.9.0 release.

Reply:
Agreed, this PR looks good for 3.9.0, and we've a year to fix it properly for 3.10.0.

---

Something like this?
```pycon
>>> from packaging import version  # pip install packaging
>>> def parse_version(version_string):
...     return version.parse(version_string.split()[1])
...
>>> parse_version("Python 3.10") >= parse_version("Python 3.9")
True
>>> parse_version("Python 3.10") <= parse_version("Python 3.9")
False
>>> parse_version("Python 3.10") == parse_version("Python 3.10")
True
>>>

Reply:
Oh, that looks like a good approach.  I forgot about packaging.

Reply:
And rather than the `.split()[1]`, looks like we could use `r.get_version()`:

```python
def parse_version(version_string):
     return version.parse(version_string)
```

https://github.com/python/pythondotorg/blob/0f07e9daf14e9f45853b61b7e9724c3741870762/downloads/models.py#L121-L125

https://github.com/python/pythondotorg/blob/0f07e9daf14e9f45853b61b7e9724c3741870762/downloads/tests/test_models.py#L63-L66